modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -16,6 +16,33 @@ New features are added to the language continuously, and occasionally, some feat
 This section lists all of the features that have been removed, deprecated, added, or extended in different Cypher versions.
 Replacement syntax for deprecated and removed features are also indicated.
 
+[[cypher-deprecations-additions-removals-5.9]]
+== Version 5.9
+
+=== New features
+
+[cols="2", options="header"]
+|===
+| Feature
+| Details
+
+a|
+label:functionality[]
+label:new[]
+
+[source, syntax, role=noheader]
+----
+quantifiedPathPattern ::= 
+  "(" fixedPath [ "WHERE" booleanExpression ] ")" quantifier
+
+fixedPath ::= nodePattern [ relationshipPattern nodePattern ]+
```

Comment:
```suggestion
MATCH ((x:A)-[:R]->(z:B WHERE z.h > x.h)){1,5}
```

I see two styles on this page, an example-based one and a syntax-based one. The latter only seems to work if it is self-contained. Would it be better to have an example of a MATCH clause? This was also what was done for label expressions.